src/index.html

```diff
@@ -12,12 +12,14 @@
   </head>
   <body>
     <header class="header">
-      <img
-        alt="logo"
-        href="#Home"
-        class="logo"
-        src="images/logo.png"
-      >
+      <a>
```

Comment:
you forgot to move `href` attribute to `a` tag :)
Please, fix this last point to make it perfect)